README.md

```diff
@@ -120,14 +120,6 @@ More formal benchmarks are on their way.
 
 ## Integrations
 
-### SVG
```

Comment:
We should probably mention [`vello_svg`](https://github.com/linebender/vello_svg/) here. Maybe that should be a follow-up PR?

Reply:
Now that it's published I'll add it back in and point to the repo. :+1: